arc/sandbox/engine/simulator.py

```
@@ -72,6 +72,42 @@ def evaluate_single_scenario(
+    secrets=[modal.Secret.from_name("openai-secret")],
+    timeout=300,
+    min_containers=1,
+    max_containers=50,
+    scaledown_window=60,
+)
+@modal.fastapi_endpoint(method="POST")
+def evaluate_scenario_endpoint(request_data: dict) -> dict[str, Any]:
+    """
+    Web endpoint version of evaluate_single_scenario.
+    Can be called via HTTP without Modal authentication.
+    
```

Ruff (0.11.9)

Blank line contains whitespace

Remove whitespace from blank line

(W293)